### ISSUE TITLE
Neutrinojs web stack has "Invalid Host/origin header" issues

### ISSUE THREAD

**Issue by bingomanatee** [neutrinojs/neutrino#1270]
### Bug or issue?
https://stackoverflow.com/questions/53906317/react-project-for-neutrinojs-displays-invalid-host-origin-header-in-console
Please try to answer the following questions:

- What version of Neutrino are you using? 9 / @next
- Are you trying to use any presets? If so, which ones, and what versions?
- Are you using the Yarn client or the npm client? What version? Yarn 1.12
- What version of Node.js are you using? 10.10
- What operating system are you using? OSX
- What did you do? Created a stock web project
- What did you expect to happen?
- What actually happened, contrary to your expectations?
repeated console messages:
```
"[WDS] Disconnected!"
"Invalid Host/Origin header" 
client?92e7:177 [WDS] Disconnected!
close @ client?92e7:177
onclose @ socket.js?e29c:17
EventTarget.dispatchEvent @ sockjs.js?9be2:170
(anonymous) @ sockjs.js?9be2:969
setTimeout (async)
SockJS._close @ sockjs.js?9be2:957
SockJS._transportMessage @ sockjs.js?9be2:897
EventEmitter.emit @ sockjs.js?9be2:86
WebSocketTransport.ws.onmessage @ sockjs.js?9be2:2961
client?92e7:174 Invalid Host/Origin header
```
The web client spits this continuously. 

<img width="1148" alt="screen shot 2018-12-23 at 10 49 36 am" src="https://user-images.githubusercontent.com/684016/50386740-8ad18900-06a1-11e9-8d2f-7bd03b30bb03.png">

**Comment by dantman** [comment]
Same issue, also Neutrino v9, here's a few snippets from `neutrino --inspect`.
```js
  target: 'web',
  context: '/Users/daniel/Workspace/work/GameFace/azure-gameface-webapp',
// ...
  output: {
    path: '/Users/daniel/Workspace/work/GameFace/azure-gameface-webapp/build',
    publicPath: '/',
    filename: 'assets/[name].[contenthash:8].js'
  },
// ...
  entry: {
    index: [
      '/Users/daniel/Workspace/work/GameFace/azure-gameface-webapp/src/index'
    ]
  }
```

**Comment by dantman** [comment]
Using `--allowed-hosts localhost:5000` or `--allowed-hosts localhost` doesn't fix the issue. Though `--disable-host-check` works around it, but it's not a solution.

**Comment by edmorley** [comment]
Hi! 

This is due to a regression in webpack-dev-server 3.1.11 rather than something in Neutrino. 

See here for their explanation:
https://github.com/webpack/webpack-dev-server/issues/1604

They say the issue is fixed in webpack-dev-server 3.1.14, though there is mention of a fix in sock-js too, so I would just delete `node_modules` and your lockfile and reinstall using yarn/npm to ensure you have the latest version of subdependencies.

**Comment by edmorley** [comment]
Closing since fixed in webpack-dev-server 3.1.14.
